github/Requester.py

```diff
@@ -447,6 +447,20 @@ def encode(local_path):
             headers["Content-Length"] = str(os.path.getsize(input))
         return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
 
+    def requestMemoryBlobAndCheck(
+        self, verb, url, parameters, headers, file_like, cnx=None
+    ):
+        def encode(file):
+            return headers["Content-Type"], file_like
```

Comment:
Okay, last question, I swear! Why does this method take file as a parameter and then use file_like from the outer function (and not what was passed in)? What am I missing?

Reply:
When I wrote it I didn't check the requirements on `encode` are - i.e. didn't follow it to where it was used - so I just made sure that it has the same signature as `encode` in `requestBlob`. I can make the parameter an underscore and add a comment to explain that it's ignored and why. 

Alternatively briefly looking at what `__requestEncode` does I could change the logic there, but I feel like what's currently done is less invasive.

Reply:
@s-t-e-v-e-n-k